src/types/json.h

```diff
@@ -217,6 +217,29 @@ struct JsonValue {
     return results;
   }
 
+  StatusOr<std::vector<size_t>> GetBytes(std::string_view path, JsonStorageFormat format,
+                                         int max_nesting_depth = std::numeric_limits<int>::max()) const {
+    std::vector<size_t> results;
+    Status s;
+    try {
+      jsoncons::jsonpath::json_query(value, path, [&](const std::string & /*path*/, const jsoncons::json &origin) {
```

Comment:
```suggestion
      jsoncons::jsonpath::json_query(value, path, [&](const std::string & /*path*/, const jsoncons::json &origin) {
        if (!s) return;
```

Reply:
There's no initialization failure.

We should check it since the lambda function can be executed more than once.